tokens/xnl.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0x06a00715e6f92210af9d7680b584931faf71a833",
+    "aurora_address": "0x06a00715e6f92210af9d7680b584931faf71a833",
```

Comment:
I don't think that you have a corresponding address in Aurora. Even if you have, the address would be definitely different. So please just remove that field.